util/config/makeSysBasicTypes

```diff
@@ -48,7 +48,9 @@ END
 "UCHAR_MAX", "c_uchar",
 "SHRT_MAX", "c_short",
 "USHRT_MAX", "c_ushort",
-#"UINTPTR_MAX" => "c_void_ptr", # comment out if you want pointer opaque so no ptr arithmetic
+"INTPTR_MAX" => "c_intptr",
+"UINTPTR_MAX" => "c_uintptr",
+"PTRDIFF_MAX" => "c_ptrdiff",
```

Comment:
I'm not at all a Perl programmer, but it surprised me that => is used for the new entries and "," for the old ones.  Even if both are legal, can we unify on one (ideally comma to minimize the diff)?